src/java/fr/paris/lutece/portal/service/scheduler/JobSchedulerService.java

```diff
@@ -67,7 +67,7 @@ public static JobSchedulerService getInstance( )
     {
         if ( _singleton == null )
         {
-            synchronized( JobSchedulerService.class )
+            synchronized ( JobSchedulerService.class )

```

Comment:
Here again for synchronisation scope

Reply:
Corrected in another commit of the same PR.